app/build.gradle.kts

```diff
@@ -33,4 +33,6 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib:_")
     implementation("androidx.constraintlayout:constraintlayout:_")
     implementation("androidx.appcompat:appcompat:_")
+    implementation("androidx.emoji2:emoji2:_")
+    implementation("androidx.emoji2:emoji2-views:_")
```

Comment:
❓ I think these dependencies are not needed if `EmojiTextView` is switched to `TextView`, correct?